src/index.html

```diff
@@ -8,7 +8,14 @@
   </head>
   <body>
     <header>
-      <h1>JEOPAR-SEA!</h1>
+      <button class="leaderboard-button" type="button" name="button">Leaderboard</button>
+      <div class="dropdown-menu hide">
+        <h2 class="winner-board-header">Global Leader Board</h1>
+        <ol class="winners-list">
+        </ol>
+      </div>
+      <h1 class="main-header-heading">JEOPAR-SEA!</h1>
```

Comment:
I think we can probs get rid of this class name since we only have 1 h1

Reply:
Agreed!